src/resourceIndex.js

```diff
@@ -128,8 +128,51 @@ async function updateResourceVersions() {
   resources = JSON.parse(fileContents);
 }
 
+/**
+ * XXX FIXME - class is a stub
+ */
+class ListResources {
+  constructor() {
+    // process query params here?
+  }
+  get data() {
+
+    /**
+     * TODO - don't hardcode these, use our routing rules instead. TODO XXX
+     */
+    const skipFirstWords = [
```

Comment:
I just deleted the following from the `nextstrain-data` bucket:
- `zika_test.json`
- `zika-tutorial-metadata-via-api.json`
- `zika-v2`

I couldn't find any JSONs in `nextstrain-data` bucket that began with:
- `ncovRegional`
- `ncov-country`
- `ncov-priorities`
- `ncov-with-priorities`
- `ncovRefocus`
- `ncov-test`
- `seattle`

Reply:
> I couldn't find any JSONs in `nextstrain-data` bucket that began with: `ncovRegional` [...]

If you toggle "Show versions" we can see what's going on here:

<img width="504" alt="image" src="https://github.com/nextstrain/nextstrain.org/assets/8350992/f40ca1b8-b718-4188-919c-7fac853597db">

With [how I handle delete markers](https://docs.nextstrain.org/en/latest/guides/snapshots.html#s3-delete-markers) I'm still exposing all the (~600) versions prior to the one that the delete marker "deleted". These datasets will be hidden from the UI once I import the routing rules into the API which hands us resource-index data, so it won't be an issue for the UI. But if we really want to clean up the bucket we'll have to consider old versions as well. (I don't have strong feelings on whether this is worth it.)

Reply:
Got it. Thank you! This was my same confusion on Slack with the https://nextstrain.org/ncov/north-america/fft@2021-01-13. I do think the indexer is doing the correct thing here. I've just gone through and deleted past versions of:
- `zika-test`
- `zika-tutorial-metadata-via-api`
- `zika-v2`
- `ncovRegional`
- `ncov-country`
- `ncov-priorities`
- `ncov-with-priorities`
- `ncovRefocus`
- `ncov-test`
- `seattle`